.github/workflows/benchmark-command.yml

```diff
@@ -28,7 +28,7 @@ on:
           - good
           - medium
           - dev
-          - master_medium
+          - master_best
```

Comment:
Aren't the `best` on line 26 already benchmark against master?

Reply:
That `best` preset is for Tabular.
The `master_best` preset is for AutoMM

More on that here: https://github.com/autogluon/autogluon/blob/master/CI/bench/multimodal/custom_user_dir/multimodal_frameworks_template.yaml
https://github.com/autogluon/autogluon-bench/blob/master/sample_configs/resources/multimodal_frameworks.yaml

I changed it to `master_best` because we were benchmarking on `medium_quality` earlier, best will give better results

Reply:
The current name is somewhat confusing. It would be beneficial to rename it, ensuring that its purpose is distinctly clear to all team members.

Reply:
Yup makes sense, can I put a follow-up PR because I have already started a benchmark run on this.
Touching framework/preset names means restarting all the benchmarking tasks.